src/parsers/directive.parser.ts

```diff
@@ -17,24 +17,24 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		let results: string[] = [];
 
 		template = this._normalizeTemplateAttributes(template);
-		$(template)
```

Comment:
I found a more elegant way of fixing the bug regarding parent template: https://github.com/biesbjerg/ng2-translate-extract/blob/master/src/parsers/directive.parser.ts#L22